lib/Extension/LanguageServerPhpCsFixer/Provider/PhpCsFixerDiagnosticsProvider.php

```diff
@@ -243,8 +243,6 @@ private function createRangesForDiff(Diff $fileDiff): array
 
                     continue;
                 }
-
-                throw new LogicException('This is not intended to be reached');
```

Comment:
cc @lumnn this _was_ reached of course and crashes the diagnostic engine :smile:

Reply:
Bit late to answer, any chances to share a piece of code on which this would be reached?

Reply:
No worries, will have a look into that then